release_dashboard/lib/services/local_conductor.dart

```diff
@@ -83,4 +95,16 @@ class LocalConductorService extends ConductorService {
     );
     return startContext.run();
   }
+
+  @override
+  Future<void> conductorNext(BuildContext context) async {
+    final ReleaseDashboardNextContext nextContext = ReleaseDashboardNextContext(
+      autoAccept: false,
```

Comment:
We never want to autoAccept or force in the release dashboard

Reply:
FYI we will eventually have to implement a UI for force.

Reply:
Right, added a todo.